### PR TITLE
Feature/register screen UI

### PR DESCRIPTION
**Pull request by gracenudze**

### CODE REVIEW

**Comment by Knexis** [comment]
You did a great job on this!

image

**Comment by Knexis** [comment]
Put some margin between the First name input box and the curve 
image

**Comment by Knexis** [comment]
Put some margin between the First name input box and the curve 
image

File: app/src/main/java/com/example/firstapplication/MainActivity.java [hunk 1]
```diff
@@ -18,25 +18,28 @@ protected void onCreate(Bundle savedInstanceState) {
     }
```
Comment (Knexis) [inline app/src/main/java/com/example/firstapplication/MainActivity.java:18]: The purpose of this class has changed. Let's rename to `RegisterUserActivity`. Same applies to it's layout: `activity_register_user`

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -4,59 +4,163 @@
     xmlns:tools="http://schemas.android.com/tools"
```
Comment (Knexis) [inline app/src/main/res/layout/activity_main.xml:4]: Let's change all EditText views into TextInputLayouts

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -4,59 +4,163 @@
...
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="100dp"
-        android:ems="10"
-        android:hint="Username"
-        android:textStyle="bold|italic"
-        android:inputType="textPersonName"
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/etFirstName"
+        android:layout_width="322dp"
```
Comment (Knexis) [inline app/src/main/res/layout/activity_main.xml:12]: As much as possible try not to give fixed sizes to views. Let's change the fixed width to `match_parent`

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -4,59 +4,163 @@
...
+        android:hint="First Name"
+        app:hintTextAppearance="@style/FloatingHintStyle"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.325"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="335dp"
```
Comment (Knexis) [inline app/src/main/res/layout/activity_main.xml:23]: Let's change the fixed width to match_parent

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -4,59 +4,163 @@
...
+        app:layout_constraintStart_toStartOf="@+id/etFirstName"
+        app:layout_constraintTop_toBottomOf="@+id/etFirstName" />
 
     <EditText
-        android:id="@+id/etPassword"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="24dp"
+        android:id="@+id/etPhone"
+        android:layout_width="331dp"
```
Comment (Knexis) [inline app/src/main/res/layout/activity_main.xml:45]: Let's change the fixed width to match_parent

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -4,59 +4,163 @@
...
+        android:hint="Phone Number"
+        android:inputType="phone"
+        app:layout_constraintEnd_toEndOf="@id/etLastName"
+        app:layout_constraintHorizontal_bias="0.2"
+        app:layout_constraintStart_toStartOf="@id/etLastName"
+        app:layout_constraintTop_toBottomOf="@+id/etLastName" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/etPassword"
+        android:layout_width="360dp"
```
Comment (Knexis) [inline app/src/main/res/layout/activity_main.xml:58]: Let's change the fixed width to match_parent

File: app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -4,59 +4,163 @@
...
+        app:hintTextAppearance="@style/TextAppearance.AppCompat.Small"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintLeft_toLeftOf="@id/etPhone"
+        app:layout_constraintRight_toRightOf="@id/etPhone"
+        app:layout_constraintTop_toBottomOf="@id/etPhone"
+        app:passwordToggleEnabled="true"
+        app:passwordToggleTint="#808080">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="332dp"
```
Comment (Knexis) [inline app/src/main/res/layout/activity_main.xml:70]: Let's change the fixed width to match_parent

File: app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -4,59 +4,163 @@
...
+            android:layout_height="48dp"
+            android:ems="15"
+            android:inputType="textPassword"
+            android:textStyle="bold" />
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/etConfirmPassword"
+        android:layout_width="360dp"
```
Comment (Knexis) [inline app/src/main/res/layout/activity_main.xml:79]: Let's change the fixed width to match_parent

File: app/src/main/res/values/styles.xml [hunk 9]
```diff
@@ -1,10 +1,14 @@
...
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <!-- Customize your theme here. -->
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="FloatingHintStyle" parent="@style/TextAppearance.AppCompat">
+        <item name="android:textColor">@color/black</item>
```
Comment (Knexis) [inline app/src/main/res/values/styles.xml:11]: Change color from black to this 
image

**Review by Knexis — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/firstapplication/MainActivity.java (hunk 1 above)
Comment (gracenudze) [reply app/src/main/java/com/example/firstapplication/MainActivity.java:18]: okay will do that

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (gracenudze) [reply app/src/main/res/layout/activity_main.xml:4]: yes please will do that

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (gracenudze) [reply app/src/main/res/layout/activity_main.xml:12]: alright sure

File: app/src/main/res/values/styles.xml (hunk 9 above)
Comment (gracenudze) [reply app/src/main/res/values/styles.xml:11]: okay will do that

**Comment by gracenudze** [comment]
> You did a great job on this!
> 
> image

Thank you very much!

**Comment by Knexis** [comment]
> Put some margin between the First name input box and the curve image

@gracenudze  you missed this

**Pull request merged**